oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/AsyncIndexUpdate.java

```diff
@@ -501,9 +501,9 @@ private void runWhenPermitted() {
             long currentTime = System.currentTimeMillis();
             if (leaseEndTime > currentTime) {
                 long leaseExpMsg = (leaseEndTime - currentTime) / 1000;
-                String err = String.format("Another copy of the index update is already running; skipping this update. " +
+                String msg = String.format("Another copy of the index update is already running; skipping this update. " +
                         "Time left for lease to expire %d s. Indexing can resume by %tT", leaseExpMsg, leaseEndTime);
-                indexStats.failed(new Exception(err, newConcurrentUpdateException()));
```

Comment:
@thomasmueller  - I don't think so, because here we actually mark the index lane as failing in case there is a concurrent update running (I suppose this was a choice), in case we just log an info and don't mark these stats as failing and simply return, it shouldn't have any side effects.

Basically choosing here if we want to mark the indexing cycle as failed or simply skip it till next run.